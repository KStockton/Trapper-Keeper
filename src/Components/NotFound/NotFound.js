import React from "react";
import "./NotFound.scss";
import { Link } from "react-router-dom";

const NotFound = () => (
  <div className="not-found-component">
    <img
      src="https://i.imgur.com/aJoxMYW.jpg"
      alt="404, page not found"
      className="not-found"
    />
    <Link to="/">
      <button className="home-btn" onClick={console.log('HEYYYYY')}>Home Page</button>
    </Link>
  </div>
);

export default NotFound;
