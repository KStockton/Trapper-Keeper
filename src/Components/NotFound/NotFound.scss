.notFoundComponent {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-evenly;
  height: 100vh;
}

.notFound {
  margin: 120px auto 0 auto;
  width: 50%;
}

.home-btn {
  height: 80px;
  width: 300px;
  background-color: #448DF5;
  color: #C7E2FF;
  border-radius: 5px;
  border-color: #C7E2FF;

  &:focus {
    outline: none;
  }
}