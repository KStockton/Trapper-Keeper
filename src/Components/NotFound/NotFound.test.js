import React from "react";
import { shallow } from "enzyme";
import NotFound from "./NotFound";

describe("NotFound", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<NotFound />);
  });
  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
