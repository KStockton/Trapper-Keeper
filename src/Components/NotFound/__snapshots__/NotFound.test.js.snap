// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should match snapshot 1`] = `ShallowWrapper {}`;
